vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
```

Comment:
should we set the `globals` config so that we don't have to import test functions everywhere similar to jest? 

```
// vitest.config.ts
import { defineConfig } from 'vitest/config'

export default defineConfig({
  test: {
    globals: true,
  },
})
```

If we set this, then we just need to add this to `tsconfig.json` as well:

```
// tsconfig.json
{
  "compilerOptions": {
    "types": ["vitest/globals"]
  }
}
```

Reply:
Good idea, will change it :)